templates/buildkite-elastic.yml

```diff
@@ -126,6 +127,11 @@ Parameters:
     Default: t2.nano
     MinLength: 1
 
+  EBSOptimized:
+    Description: Optional - Enabled EBSOptimized for the instance
```

Comment:
>  that don't have it automatically enabled" ?

What happens if you specify `EBSOptimized: false` for instances that get it by default? are they unoptimized?